.circleci/config.yml

```diff
@@ -134,6 +135,30 @@ jobs:
           paths:
             - ./build/bin/inspect
 
+  docs:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /hpx
+      - run:
+          name: Building Sphinx Documentation
+          command: |
+              ninja docs
+      - persist_to_workspace:
+          root: /hpx
+          paths:
+            - ./build
+
+  docs_push:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: /hpx
+      - run:
+          name: Pushing Sphinx Documentation
+          command: |
+              ninja git_docs
```

Comment:
I think this should only happen on the master branch, otherwise, unstable will be overriden by PRs

Reply:
Yeah, that still needs to be added before merging. Added a todo item in the PR description.